cmd/pcr0tool/commands/diff/command.go

```diff
@@ -155,12 +156,21 @@ func (cmd Command) Execute(args []string) {
 	assertNoError(err)
 
 	var regs registers.Registers
-	if len(*cmd.registers) > 0 {
+	switch *cmd.registers {
+	case "":
```

Comment:
I thought that fallthrough is not a by-default behaviour in golang

Reply:
It will not fallthrough -- it was intentionally. To do not mislead the end user, we do not extract local register until it was explicitly asked by the user. Otherwise the end user might exec the tool on their laptop and do not understand why the PCR0 is not the same in `tpm2_pcrread` from a production host.

But I guess I need to leave a comment in code. Will do.

Reply:
Fixed.